spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'support/shared_contexts/shared_authenticated.rb'
```

Comment:
Add `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }` to rails helper and remove this line

Reply:
Done!